apis/projectcontour/v1/httpproxy.go

```diff
@@ -551,6 +551,10 @@ type Route struct {
 	// +optional
 	PathRewritePolicy *PathRewritePolicy `json:"pathRewritePolicy,omitempty"`
 	// The policy for managing request headers during proxying.
+	// NOTE: You can set the custom values to the host header on a request using
+	// the below format "%REQ(X-Header-Namee)%". If the value of the header
```

Comment:
nit

```suggestion
	// the below format "%REQ(X-Header-Name)%". If the value of the header
```

Reply:
Also maybe add a note similar to the Envoy docs about taking consideration for security implications when this is in use: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-field-config-route-v3-routeaction-host-rewrite-header

Off the top of my head, I'm not quite sure how this works when auth support or another filter is also configured...

Reply:
addressed in https://github.com/projectcontour/contour/pull/5678/commits/25fcc214f366a04d05dae9396774ffc8afbfb700